templates/ubuntu/tpl_ceph_remove.go

```diff
@@ -68,6 +71,7 @@ func (m *UbuntuCephRemoveTemplate) Render(pkg urknall.Package) {
 	)
 	pkg.AddCommands("remove",
 		Shell("rm -r /var/lib/ceph/"),
+		Shell("rm -r /home/megdc/ceph-cluster"),
```

Comment:
@thomasalrin Good catch.

@rajthilakmca  Actually its passed in as a variable i think.